05.wc/wc.rb

```diff
@@ -0,0 +1,78 @@
+# ! /usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+SEPARATOR_SIZE = 1
+STDIN_FILENAME = '-'
+TOTAL = '合計'
+
+def main
+  options = parse_options(ARGV)
+  counts = []
+  ARGF.each(nil) do |input_text|
+    count = {}
+
+    count[:filename] = ARGF.filename
+    count[:line] = input_text.lines.count if options[:l]
+    count[:word] = input_text.split(/\s+/).size if options[:w]
+    count[:bytesize] = input_text.size if options[:c]
+    counts << count
+  end
+  counts << calculate_counts_total(counts) if counts.size > 1
+  max_column_widths = calculate_wc_max_column_widths(counts)
```

Comment:
```suggestion
  max_widths = calculate_max_widths(counts)
```

修飾語は省略できそうです

https://bootcamp.fjord.jp/pages/avoid-too-specific-var-name